andela/activo-api
# #160090098 Fix  asset category update issue

#### What does this PR do?
- This task fixes a bug that causes asset category custom attributes not to change when updated via the `/api/v1/asset-categories`

    **Current Behavior**
    When the custom attributes of an asset category is updated via the `/api/v1/asset-categories` endpoint, the custom attributes don't get updated in the database.

    **Expected Behavior**
    An asset category's custom attributes should be successfully updated in the database when a PATCH request is sent to `/api/v1/asset-categories` endpoint.

#### Description of Task to be completed?
- Fix the issue causing custom attributes changes not reflecting in the database

#### How should this be manually tested?
* Fetch the branch: `git fetch bg-fix-asset-category-update-issue-160090098`
* Checkout to the branch: `git checkout bg-fix-asset-category-update-issue-160090098`
* Switch to the app environment: `pipenv shell` (you need `zsh`, `bash` or `sh` for this)
* Seed the database: `flask seed_database`
* Run the application: `flask run`
* Open `postman` and try sending a PATCH request to `/api/v1/asset-categories`
* You will see the updated changes in the response body
* **Sample Request Body**
    ```json
    {
        "name": "Apple Hello",
        "customAttributes": [
            {
                "id": "-LL-4Vza6DqSwo2DF-9o",
                "inputControl": "text",
                "isRequired": false,
                "key": "warranty",
                "label": "Warranty"
            }
        ]
    }
    ```

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
[#160066284](https://www.pivotaltracker.com/story/show/160066284)

#### Postman collection:
- N/A

#### Screenshot
- N/A
